### ISSUE TITLE
Vulnerability of dependency "golang.org/x/text"

### ISSUE THREAD

**Issue by Silence-worker-02** [spf13/afero#394]
Hello, we are a team researching the dependency management mechanism of Golang. During our analysis, we came across your project and noticed that it contains a vulnerability (CVE-2022-32149). In your project, the golang.org/x/text package is being used at version golang.org/x/text v0.3.7, but the patched version is v0.3.8. To fix the vulnerability, we recommend modifying the go.mod file to update the version to v0.3.8 or higher. Thank you for your attention to this matter.

**Comment by jxsl13** [comment]
if this library does not bump its dependency, you could do at least a hotfix and manually bump that indirect dependency to 0.3.8

**Comment by pjonsson** [comment]
@Silence-worker-02 is this a duplicate of #381?

**Comment by Silence-worker-02** [comment]
Yeah, but now the question is still exist. In the latest commit, go.mod still used golang.org/x/text 0.3.7.
